libs/dal/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.14.0",
     "@aws-sdk/s3-request-presigner": "^3.14.0",
+    "@nestjs/common": "^8.4.0",
```

Comment:
I think it's better אישא the DAL won't be aware nestjs. I would just replicate the exceptions in notifire/shared instead of importing nestjs. 

This is kind of architectural boundary I want to keep for now

Reply:
Good thinking I will update it.